fe/fe-core/src/main/java/com/starrocks/statistic/StatisticUtils.java

```diff
@@ -55,8 +55,7 @@ public static ConnectContext buildConnectContext() {
         context.getSessionVariable().setParallelExecInstanceNum(parallel);
         context.getSessionVariable().setPipelineDop(1);
```

Comment:
should change the PipelineDop to getStatisticCollectParallelism.

Reply:
> should change the PipelineDop to getStatisticCollectParallelism.

fixed. I change pipeline_dop to getStatisticCollectParallelism and parallel_exec_instance_num to 1 so that we use pipeline internal parallelism